src/base/orgs/TransferOwnership.svelte

```diff
@@ -98,7 +108,7 @@
 
     <div slot="body">
       {#if state == State.Idle}
-        <input type="text" size="40" disabled={state !== State.Idle} bind:this={input} bind:value={newOwner} />
+        <input type="text" size="40" disabled={state !== State.Idle} placeholder="Enter Ethereum address or ENS subdomain" bind:this={input} bind:value={newOwner} />
```

Comment:
TBH I'm not a fan of this change as it's not generalized across address inputs and not really thought through, eg. it only supports .radicle.eth domains. I would revert this to the previous behavior and we can look at the problem more comprehensively.